src/Dockerfile.usefulpackages

```diff
@@ -1,4 +1,4 @@
-LABEL maintainer="Christoph Schranz <christoph.schranz@salzburgresearch.at>"
+LABEL authors="Christoph Schranz <christoph.schranz@salzburgresearch.at>, Mathematical Michael <consistentbayes@gmail.com>"
```

Comment:
i read maintainers is going to be deprecated in favor of this syntax

Reply:
I've read that too. And thanks for beeing the co-author! ;)